docs/pre-release/releaseNotes.yml

```diff
@@ -33,6 +33,34 @@ items:
   - version: 2.y.z
     date: 'TBD'
     notes:
+      # Headliners
+      - type: feature
+        title: Traffic-manager Helm Chart
+        body: >-
+          We now support installing the traffic-manager via helm.
+          This will make it easier for operators to see what the
+          traffic-manager needs to install so that telepresence works
+          correctly, as well as providing a more direct route to customizing
+          installation.
+        image: ./telepresence-2.3.3-helm.png
```

Comment:
Am I right that that image exists on a branch in getambassador.io.git?  Please copy it to telepresence.io.git `docs/pre-release/release-notes/`

Reply:
ah yes, good point, forgot to bring it over when i copied Donny's changes into this branch